pkg/assembler/clients/helpers/bulk.go

```diff
@@ -30,6 +30,7 @@ import (
 func GetBulkAssembler(ctx context.Context, gqlclient graphql.Client) func([]assembler.AssemblerInput) error {
 	logger := logging.FromContext(ctx)
 	return func(preds []assembler.IngestPredicates) error {
+		var rvErr error
```

Comment:
Should we make this a list since it's set up in a for loop?

Reply:
So I was deciding weather to leave the existing `logger.Errorf` calls, if we capture all the errors and combine them, then we don't need the logs.

If we leave as is, we can log on (multiple possibly) errors, and make sure the command returns at least one error up the stack